buildconfig/stubs/pygame/.flake8

```diff
@@ -16,10 +16,11 @@
 #       W504 line break after binary operator
 
 [flake8]
-ignore = F401, F403, F405, F811, E301, E302, E305, E501, E701, E704, E741, B303, W504
+ignore = B303, E301, E302, E305, E501, E701, E704, E741, F401, F403, F405, F811, W504
 # We are checking with Python 3 but many of the stubs are Python 2 stubs.
 # A nice future improvement would be to provide separate .flake8
 # configurations for Python 2 and Python 3 files.
```

Comment:
these comments are rather outdated, this PR would be a good opportunity to clean up these :smile: